### PR TITLE
Update queries to reflect LDES project params

### PR DESCRIPTION
**Pull request by jszinai**
- Updated query related to generation_project_info to use variable_o_m_cost table and variable_o_m_cost_scenario id, default technology assumptions (outage rates, baseload status) from the generation_plant_technologies table with the generation_plant_technologies_scenario_id
- Updated financial params to reflect $2018 dollars and 5% discount and interest rates of LDES project baseline assumptions
- Updated transmission queries to include economic multiplier as part of terrain_multiplier, and added optional params into query that creates transmission_line.csv file because that is the file actually used by SWITCH in the "build.py" transmission module
- Added a query for transmission params to use data from the transmission_base_capital_cost table with the transmission_base_capital_cost_scenario_id so the the transmission costs are not hard-coded, and reflect the updated LDES project baseline assumptions for transmission cost
- Fixed typo of missing column name for gen_can_provide_cap_reserves
- Text editor fixed trailing white spaces and converted tabs to spaces

Tested above changes with db scenario 177, which is the baseline scenario for the LDES project with a WECC-wide 0 carbon cap, and planning reserves enabled. Compared the output data to prior get_input version results with Berkeley db for same scenario.

### CODE REVIEW

File: switch_model/wecc/get_inputs.py [hunk 1]
```diff
@@ -428,54 +434,41 @@ def query_db(full_config, skip_cf):
...
          SELECT start_load_zone_id || '-' || end_load_zone_id, t1.name, t2.name,
-             trans_length_km, trans_efficiency, existing_trans_cap_mw
+             trans_length_km, trans_efficiency, existing_trans_cap_mw, transmission_line_id,
+            derating_factor, terrain_multiplier * transmission_cost_econ_multiplier as terrain_multiplier,
+            new_build_allowed
          FROM switch.transmission_lines
              join load_zone as t1 on(t1.load_zone_id=start_load_zone_id)
              join load_zone as t2 on(t2.load_zone_id=end_load_zone_id)
-         WHERE start_load_zone_id <= end_load_zone_id 	
+         WHERE start_load_zone_id < end_load_zone_id
```
Comment (pesap) [inline switch_model/wecc/get_inputs.py:402]: Is there a reason why to remove the <= sign from this?

File: switch_model/wecc/get_inputs.py [hunk 2]
```diff
@@ -648,8 +650,9 @@ def query_db(full_config, skip_cf):
     ########################################################
     # FINANCIALS
 
+    #updated from $2016 and 7%
     write_csv(
-        [[2016, 0.07, 0.07]],
+        [[2018, 0.05, 0.05]],
```
Comment (pesap) [inline switch_model/wecc/get_inputs.py:613]: Thanks for catching this!

**Review by pesap — COMMENTED** [review]
Looks good! If you attend to the comment. I can resolve the conflict and merge right away.

File: switch_model/wecc/get_inputs.py (hunk 1 above)
Comment (PatyHidalgo) [reply switch_model/wecc/get_inputs.py:402]: Yes, because there wouldn't be lines from LZ_i to LZ_i. So <= or < would be the same :)

**Review by PatyHidalgo — APPROVED** [review]

**Comment by pesap** [comment]
@jszinai , you can delete your branch :) Thank you!

**Pull request merged**